### ISSUE TITLE
[Update Request]: Drop CrystalDiskMark Version 8.0.4a & 8.0.4c and adjust 8.0.4

### ISSUE THREAD

**Issue by DenWin** [microsoft/winget-pkgs#122931]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

CrystalDewWorld.CrystalDiskMark

### Package Version

8.0.4

### Please describe the changes you would like to see

There exist a new verson `8.0.4c`, thought the sorting order of winget will assume `8.0.4` is newer than `8.0.4.c`.

Therefore the versions 8.0.4c and 8.0.4a should be dropped and only 8.0.4 kept and adjusted accordingly!

![image](https://github.com/microsoft/winget-pkgs/assets/13867326/32b68672-c21b-4797-9ae2-5bdadca3c39a)
```
8.0.4  [2021/07/11]
8.0.4a [2022/01/30]
8.0.4b [2022/07/17]
8.0.4c [2022/10/30]
```

See Issues https://github.com/microsoft/winget-pkgs/issues/122923 and https://github.com/microsoft/winget-pkgs/issues/58540

**Comment by Trenly** [comment]
See also -
* https://github.com/microsoft/winget-cli/issues/2116
* https://github.com/microsoft/winget-cli/issues/3073

**Comment by DenWin** [comment]
Hence #122942 which should fix this issue, that is particular unique to CrystalDiskMark

**Comment by DenWin** [comment]
The issue is resolved, now that the version 8.0.4c was changed to 8.0.4.c
